_authentication.md

```diff
@@ -100,6 +100,16 @@ curl https://api.uphold.com/v0/me/tokens \
   -d '{ "description": "My command line script" }'
 ```
 
+> The above command returns the following JSON:
+
+```json
+{
+    "accessToken":"c386ae9c4557c1c661b15911071b06d9e6c3fc9a",
+    "description":"token 1",
```

Comment:
Wrong description, based on the example request above.

Reply:
Fixed.